deploy/cdk/lib/foundation/foundation-stack.ts

```diff
@@ -106,6 +106,10 @@ export class FoundationStack extends Stack {
     })
 
     this.cluster = new Cluster(this, 'Cluster', { vpc: this.vpc })
+    new CfnOutput(this, 'ExportsOutputFnGetAttClusterEB0386A7Arn2F2E3C3F', {
```

Comment:
Remove this. cdk should create this for you for any stacks that reference this stack's cluster.clusterArn.

Reply:
I am not sure I can.  If I do it won't let me deploy says it can't be deleted because it is used by another stack.